### PR TITLE
Add awesomebot github action

### PR DESCRIPTION
**Pull request by unixorn**
Awesomebot will check all the links in README.md and ensure they're still alive.

Includes example (img.shields.io has been a problem on my own repos) to whitelist sites with intermittent issues so they don't block PR checks.

Use `--request-delay 1` so that it doesn't hit rate limits for sites that have multiple links pointing at them.

### CODE REVIEW

File: .github/workflows/awesomebot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: docker://dkhamsing/awesome_bot:latest
+      with:
+        args: /github/workspace/README.md --allow-dupe --allow-redirect --request-delay 1 --white-list https://github,https://img.shields.io,http://127.0.0.1:8080
```
Comment (AkihiroSuda) [inline .github/workflows/awesomebot.yml:18]: Why https://github has to be whitelisted?

File: .github/workflows/awesomebot.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```
Comment (AkihiroSuda) [inline .github/workflows/awesomebot.yml:15]: `@v2`

File: .github/workflows/awesomebot.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Check links in README.md
```
Comment (AkihiroSuda) [inline .github/workflows/awesomebot.yml:1]: We can have this in https://github.com/lima-vm/lima/blob/master/.github/workflows/test.yml

File: .github/workflows/awesomebot.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```
Comment (AkihiroSuda) [inline .github/workflows/awesomebot.yml:12]: ubuntu-20.04

File: .github/workflows/awesomebot.yml (hunk 1 above)
Comment (unixorn) [reply .github/workflows/awesomebot.yml:18]: Sorry, I missed that when I copied it from one of my other repos

File: .github/workflows/awesomebot.yml (hunk 4 above)
Comment (unixorn) [reply .github/workflows/awesomebot.yml]: Done

File: .github/workflows/awesomebot.yml (hunk 2 above)
Comment (unixorn) [reply .github/workflows/awesomebot.yml]: Done

File: .github/workflows/awesomebot.yml (hunk 1 above)
Comment (unixorn) [reply .github/workflows/awesomebot.yml]: Removed

**Comment by AkihiroSuda** [comment]
Added a follow-up commit

**Pull request merged**